SConstruct

```diff
@@ -312,8 +326,8 @@ if not env.GetOption('clean'):
 			print("  If you're sure it's installed, try passing INCLUDEPATH=...")
 			Exit(1)
 
-	boost_versions = ['-1_54', '-1_55', '-1_56', '-1_57', '-1_58'] # This is a bit of a hack. :(
-	bundled_libs = []
+	boost_versions = ['-1_84'] # This is a bit of a hack. :(
+	suffixes = ['-mt', f'-mt-x{env["bits"]}']
```

Comment:
Again, is `-mt-x32` a valid suffix?

Reply:
As it turns out, yes. One place where it's 32 and not 86.